src/frontend/src/views/Upload/components/Metadata/components/ImportFile/parseFile.ts

```diff
@@ -117,14 +117,14 @@ function buildMetadata({ headers, row, warningMessages }: RowInfo) {
   ) {
     metadata.collectionLocation = "";
   }
-
   return metadata;
 }
 
-function convertValue(value: string) {
-  if (value === "Yes") return true;
-  if (value === "No") return false;
-
+function convertValue(key: string, value: string) {
+  if (key === "keepPrivate" || key === "submittedToGisaid") {
+    if (value.toUpperCase() === "YES") return true;
+    return false;
+  }
   return value;
```

Comment:
```suggestion
  return False;
```

Reply:
Ah yes we take that into account at line 126 now 👌 Thank you!

Reply:
this one we want to return value since this is parsing the other fields too, otherwise we would return false for public_id, private_id, etc